src/Language/ASKEE/Exposure/Interpreter.hs

```diff
@@ -146,18 +148,20 @@ data EvalRead = EvalRead
   , erLocalVars   :: Map Ident Value
   , erPutFileFn   :: EvalWriteFileFn
   , erGetFileFn   :: EvalReadFileFn
+  , erPyHandle    :: Maybe PythonHandle
```

Comment:
Ah, there's actually another wrinkle: you use `Nothing` as a default value for `erPyHandle` in the test suite.

Reply:
Ah right. I was counting on CI to run tests for me, but I guess that's not happening at the moment. I'll have to look more closely locally :)

Reply:
Er nevermind, I've confused myself

Reply:
Perhaps instead of having `Maybe Handle` here, we could just have `Handle`. Then, in the test suite, construct an `EvalRead` once using `tasty`'s [`withResource`](https://hackage.haskell.org/package/tasty-1.4.1/docs/Test-Tasty.html#v:withResource) function and pass that `EvalRead` to each test case. Then, we wouldn't have a need for `exprAssertionWithStmts'`, since `exprAssertionWithStmts` would now take an `EvalRead` as an argument.

Reply:
Yes that sounds handy